### ISSUE TITLE
Web component: Card section - Images - beginning and ending boundary for the card is not conveyed for the screen reader user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4614]
## Detailed description

Issue:
1. The beginning and ending boundary for the card is not conveyed for the screen reader user.
2. In tab / shift tab navigation, when focus reaches the link in each card, Screen reader announces a lengthy text which makes it oververbose.

Expected:
1. Provide each card within the landmark region and give appropriate label.
2. It may be possible if the entire content in the card is given in anchor tag. In this scenario, only the heading can be given in anchor tag. Note: the link text/label should be concise and convey only the relevant information.

Note: working fine on react version.

**Comment by RobertaJHahn** [comment]
@ariellalgilmore Added Ari to issue (#4613 ) as the same solution should be implemented for all Card variants.  Once the fix is done, Ari, can you check to be sure that the fix addresses this bug and move to Done.  Thanks.

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested this on both JAWS and Voiceover on the [ link ]( https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/6642/index.html?path=/story/overview-getting-started--page)  - Working as expected.
